service/device/management/packages/job/src/main/java/org/eclipse/kapua/service/device/management/packages/job/DevicePackageUninstallTargetProcessor.java

```diff
@@ -85,4 +86,13 @@ protected JobTargetStatus getCompletedStatus(JobTarget jobTarget) {
 
         return super.getCompletedStatus(jobTarget);
     }
+
+    @Override
+    protected String getTargetDisplayName(JobTarget jobTarget) throws KapuaException {
```

Comment:
This isn't supposed to inherit from `AbstractDeviceTargetProcessor`?

Inheritance should be `DevicePackageUninstallTargetProcessor` > `AbstractDevicePackageTargetProcessor` > `AbstractDeviceTargetProcessor` > `AbstractTargetProcessor` > etc